examples/input_fodo.in

```diff
@@ -1,46 +1,45 @@
 ###############################################################################
 # Particle Beam(s)
 ###############################################################################
-beam.npart = 10000
+beam.npart = 100000
```

Comment:
Just a note: let's check what's a good number of the test to still run fast. 100K is probably still fine?
Just highlighting this here, since it probably was added in testing locally.

Reply:
I timed 100K particles on Cori at 7 min, 42 sec, which is slow, but I'm not sure what will count as acceptable for testing.  The number of particles needed for this test will depend on the precision we require for the matching of the second beam moments.  Also, 10K particles will almost certainly be too small when we start testing with space charge.

Reply:
What I am thinking are two kinds of tests:
- very quick ones that we can run in CI: there our limit are 2 CPU cores and ideally <20sec runtime
- convergence tests that we use, based on those, where we simply increase resolution and/or number of particles, respectively: those we can run on any machine, size and cost are free for us

I would like to build the 2nd always on the former, so that we can run "rough" tests or small (single particle?) tests very quickly for every code change, and do only repeat the convergence tests once in a while. Does that make sense?

Reply:
For the current numbers in this inputs set, CI runs 2.3sec on 2 CPU cores. That is totally fine :)